### DISCUSSION TITLE
should we find alternatives to this was-node-suite-comfyui ?

### DISCUSSION THREAD

**Discussion by tampadesignr** [WASasquatch/was-node-suite-comfyui#518] [Q&A]
12/15/2023 WAS-NS is not under active development. 

i see activity but i also see that message.
are you still fixing as is to maintain function with comfyui updates?

**Comment by WASasquatch** [comment]
I make changes when I have time. There are plenty of alternatives to
WAS-NS. Most big packs have incorporated all I have.

**Reply by tampadesignr** [reply]
i been doing this for a few months and alot of people love your stuff. hope all is well.
you are amazingly talented, you have given alot to this community.
i myself gave up on my talent for a long time but stuff like your tools have giving me new life.

thank you
